docs/design-document/03-connect/02-http-connector.md

```diff
@@ -5,12 +5,15 @@
 ### Configuration
 
 Before using HTTP source connector, you need to configure the server.
+
 - Please configure `sourceEnable` to `true` in `/resource/server-config.yml` to enable source functionality.
-- Please configure the source connector in `/resource/source-config.yml`, only the configuration under `connectorConfig` is described here:
-  - `connectorName`, name of the connector.
-  - (required) `path`, path of the API.
-  - (required) `port`, port of the API.
-  - `idleTimeout`, idle TCP connection timeout in seconds. A connection will timeout and be closed if no data is received nor sent within the `idleTimeout` seconds. The default is 0, which means don't timeout.
+- Please configure the source connector in `/resource/source-config.yml`, only the configuration under `connectorConfig`
+  is described here:
+    - `connectorName`, name of the connector.
+    - (required) `path`, path of the API.
+    - (required) `port`, port of the API.
+    - `idleTimeout`, idle TCP connection timeout in seconds. A connection will timeout and be closed if no data is
+      received nor sent within the `idleTimeout` seconds. The default is 0, which means don't timeout.
```

Comment:
intended line-break?

Reply:
This is the result of global formatting and doesn't seem to affect the presentation, does it need to be one line?

Reply:
Yes, it should be an unordered list here.

Reply:
No I think.

Reply:
Resolved at https://github.com/apache/eventmesh-site/pull/221/commits/49ec87ea9eac8afa45f931d7f102f0fbdc7c5646